gradio/inputs.py

```diff
@@ -63,6 +63,11 @@ def get_shortcut_implementations(cls):
         """
         return {}
 
+    def rebuild_flagged(self, dir, msg):
+        """
+        All interfaces should define a method that rebuilds the flagged input when it's passed back (i.e. rebuilds image from base64)
+        """
+        pass
```

Comment:
Maybe it makes sense to define a default method? Or are you planning on implementing this method for every AbstractInput and AbstractOutput?

Reply:
There is a default method, you are commenting on an outdated commit. By default the `data` is returned with no processing. (Like with Text)